src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -38,6 +38,7 @@
 // expects lowercase, and a native compile on windows, whether via msvc
 // or mingw-w64 addresses files in a case insensitive manner.
 #include <dbt.h>
+#include <limits>
```

Comment:
It looks like this is not needed.

Reply:
Thank you yes that's right. I previously used that head but forgot to remove it.